pkg/suites/che/workspaces_tests.go

```diff
@@ -32,13 +38,13 @@ var _ =  KubeDescribe( "[Workspaces]" , func() {
 		_ = ctrl.RunWorkspace(file, workspaceStack)
 	})
 
-	ginkgo.It("Start Python 3.7 Workspace", func() {
-		workspaceStack := "python"
+	ginkgo.It("Start NodeJS Workspace", func() {
+		workspaceStack := "nodejs"
 		httpClient, err := client.NewHttpClient()
 
```

Comment:
In Java Maven test, there is a sleep command. Can you please explain, why it is missing for the NodeJS test?

Reply:
Oh, Good catch! I was doing some tests locally and I forgot to remove this sleep command.